packages/yad/SPECS/yad.spec

```diff
@@ -0,0 +1,267 @@
+Name:      yad
+Version:   0.40.0
+Release:   6%{?dist}
+Summary:   Display graphical dialogs from shell scripts or command line
+
+License:   GPLv3+
+URL:       http://sourceforge.net/projects/yad-dialog/
+Source0:   http://downloads.sourceforge.net/%{name}-dialog/%{name}-%{version}.tar.xz
+
+
+BuildRequires:  gcc
+BuildRequires:  gtk2-devel >= 2.20.1
+BuildRequires:  desktop-file-utils
+BuildRequires:  perl(XML::Parser)
+BuildRequires:  intltool >= 0.40.0
+BuildRequires:  pkgconfig
+BuildRequires:  gettext
+
```

Comment:
does yad need jq and onigurama too?

Reply:
nope, jq needs onigurama though. yad is otherwise unrelated excepting I needed both in the dumb youtube thing